lib/src/messages/call.dart

```diff
@@ -32,12 +35,12 @@ class Call implements Message {
     String uri = validateStringOrRaise(message[3], text, "uri");
 
     List<dynamic>? args;
-    if (message.length > 4) {
+    if (message.length > minLength) {
       args = validateListOrRaise(message[4], text, "args");
     }
 
     Map<String, dynamic>? kwargs;
-    if (message.length > 5) {
+    if (message.length > (minLength + 1)) {
```

Comment:
all these checks could be made
```suggestion
    if (message.length == maxLength) {
```